modules/nf-core/mitohifi/findmitoreference/main.nf

```diff
@@ -0,0 +1,36 @@
+process MITOHIFI_FINDMITOREFERENCE {
+    tag '$species'
+    label 'process_low'
+
+    // MitoHifi does not exist as conda package
```

Comment:
I've never done that before, so I'm not sure! 

However, the developers of MitoHifi write that one of the dependencies uses a different python version than the others, which is why they suggest either using their container or a conda environment+manual installation of that tool: https://github.com/marcelauliano/MitoHiFi#21-manual-installation---dependencies

Reply:
You can always ask for help on the nf-core slack or the Gitter of bioconda, I'm not that experienced in this myself

Reply:
I got in touch with some bioconda people! Unfortunately, MitoHifi can't be made available as bioconda package or biocontainer. It is essentially a mini-pipeline written in Python, and the tools it runs need different versions of Python, which can't be installed in the same Conda environment. That’s why there is instead a Docker container.

I assume this rules out having it as a nf-core module?

Reply:
Hi, thanks for asking! I think it's fine how it's done now, but I'll have to ask for a second opinion here :) (@jfy133, @maxulysse, @ewels any opinions here?)

Reply:
I think conceptually it _may_ be OK, although I'm wary of non-standard container registries. That said github container registry should be pretty stable I would assume?

@FriederikeHanssen I vaguely remember you brought up a non-conda package at some point (maybe wrong...)

Reply:
The biocontainers image contains version 2.14.2, the github image has version 3.0.0 - but maybe biocontainers is more stable...

Reply:
Hiya. I work with the MitoHifi people. 
I'll get the latest Dockerfile into BioContainers because there's a lot of new stuff in the new version. Can you hold on until that's done ?

Reply:
https://github.com/BioContainers/containers/pull/509 🤞🏼

Reply:
@verku : the latest container is now `biocontainers/mitohifi:3.0.0_cv1`

Reply:
Thank you, this is great news! I'll update the module accordingly.